avocado/core/varianter.py

```diff
@@ -438,7 +447,7 @@ def _skip_new_data_check(self, function, args):
             raise RuntimeError("Varianter already parsed, unable to execute "
                                "%s%s" % (function, args))
 
-    def data_inject(self, key, value, path=None):   # pylint: disable=E0202
+    def data_inject(self, name, key, value, path=None):   # pylint: disable=E0202
```

Comment:
you mean `add_default_param`? I will change it even though it will not fit the current scheme (`data_*`). Anyway it's only a temporary change so I'm fine with either.